app/modules/notifications/models.py

```diff
@@ -465,6 +474,12 @@ def channels_to_send(self, digest=False):
         # In future the channels to send right now will be different for digest generation
         user_prefs = UserNotificationPreferences.get_user_preferences(self.recipient)
         channels = user_prefs.get(self.message_type, {})
+        super_type = NOTIFICATION_CONFIG[self.message_type].get('subtype_of')
```

Comment:
The supertype should be additive to the leaf type so user gets email if individual_merge_all OR individual_merge_complete is set to True for email.

Reply:
implemented